docs/pages/pack/docs/features/customizing-turbopack.mdx

```diff
@@ -113,4 +113,30 @@ module.exports = {
 
 This aliases imports of the `underscore` package to the `lodash` package. In other words, `import underscore from 'underscore'` will load the `lodash` module instead of `underscore`.
 
+## Resolve Extensions
+
+Through `next.config.js`, Turbopack can be configured to resolve modules with custom extensions, similar to webpack's [`resolve.extensions`](https://webpack.js.org/configuration/resolve/#resolveextensions) configuration.
+
+To configure resolve extension, use the `resolveExtensions` field in `next.config.js`:
+
+```js filename="next.config.js"
+module.exports = {
+  experimental: {
+    turbo: {
+      resolveExtensions: [
```

Comment:
I think it would be a good idea to move functionality like this to a shared field that both webpack and turbopack read from

Reply:
Either way, this needs to be added to the typescript types

Reply:
oh right, this isn't the next.js repo yet, we should really move the docs over

Reply:
The original comment still stands though, maybe this should be something like `experimental.compiler.resolveExtensions`?